src/s2mMuscleFatigueState.cpp

```diff
@@ -24,36 +24,47 @@ void s2mMuscleFatigueState::setState(double active, double fatigued, double rest
 {
     // Sanity check for active fibers
     if (active < 0){
+        resting += active;
```

Comment:
Please add the rationale in comment. I don't understand the relevance of this modification